app/models/solidus_stripe/gateway.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+require 'stripe'
+
+# rubocop:disable Lint/UnusedMethodArgument, Style/MethodCallWithoutArgsParentheses, Lint/UselessAssignment
+module SolidusStripe
+  class Gateway
```

Comment:
Regarding 189dd8746a513936412ec6181b23ddc084033dce, can you provide more details on how  Gateway, PaymentMethod, and PaymentSource were built? They seem to follow the API of SolidusPaypalCommercePlatform but their code are not exactly the same.

Reply:
It's just meant to be a placeholder for the implementation. Given the Gateway API is mandated by Solidus I felt it was ok to stub it out letting the implementation to be added in following PRs.